src/verbs.rs

```diff
@@ -309,16 +309,27 @@ pub unsafe fn ___ibv_query_port(
     }
 }
 
+#[inline]
+pub unsafe fn ibv_query_gid_table(
+    context: *mut ibv_context,
+    entries: *mut ibv_gid_entry,
```

Comment:
Maybe pass a slice of ibv_gid_entry here? As in the original inline implementation, `max_entries` is sizeof(*entries), so that we can use `entries.len()` as `max_entries`

Reply:
Got it. I just noticed the difference between `ibv_query_gid_table` and `_ibv_query_gid_table`😄